src/main/java/io/github/intoto/slsa/models/Provenance.java

```diff
@@ -25,6 +25,7 @@
    *
    * <p>MAY be unset/null if unknown, but this is DISCOURAGED.
```

Comment:
Should this be removed?

Reply:
Probably not, it illustrates to the users of the library that the field can be left unset or null, the library will just omit it.

Reply:
gotcha